tasks/orca_catalog_reporting/orca_catalog_reporting.py

```diff
@@ -156,7 +156,7 @@ def get_catalog_sql() -> text:  # pragma: no cover
     LIMIT :page_size+1
 ) as granules
 LEFT JOIN LATERAL
-    (SELECT json_agg(files) as files
+    (SELECT COALESCE(json_agg(files), '[]'::json) as files
```

Comment:
Turns out that json_agg returns `null` if no files are present. Not a major problem now, but might be once we implement deletes. This this addition, it defaults to `[]`